build/python.props

```diff
@@ -0,0 +1,5 @@
+<Project>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.Functions.PythonWorker" Version="1.0.201910045" />
```

Comment:
can you use build number instead of date?

Reply:
That is currently how I've defined the build number to be unique. Don't think there should be any issues with using this.

Reply:
across build systems we use build number generated by the CI system. Would be good to align versions across components and repos

Reply:
This **is** the build number generated by Azure DevOps by default. They use the current date. To make it unique, you can add a `rev` version in front of it, which is what I've done. Any different option in Azure DevOps at least would require manually incrementing patch versions.

Reply:
We have other repos : core tools, extensions using devops. Adding @soninaren / @ankitkumarr  to point to how we generate version for nuget packages

Reply:
I'm not sure if we need to strive for consistency here. This is also a valid way of incrementing the patch number.

Reply:
Node versions are currently hard coded and don't rely on the Azure DevOps pipeline build numbers, which is also ok: https://github.com/Azure/azure-functions-nodejs-worker/blob/dev/azure-pipelines.yml.